### PR TITLE
Ds tabbed interface

### PR DESCRIPTION
**Pull request by dylansarikas**
Pull request for photogram part 4.2

Creates links to various user views

### CODE REVIEW

File: app/views/users/_pill_bar.html.erb
Comment (APatel-AI) [inline app/views/users/_pill_bar.html.erb:1]: Nice use of partials here!

File: app/controllers/users_controller.rb
Comment (APatel-AI) [inline app/controllers/users_controller.rb:1]: Clean and Concise

**Review by APatel-AI — APPROVED** [review]
Great job Dylan! The code is consistent and easy to follow.

File: app/views/photos/_photo.html.erb [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+      <div class="card-body">
+        <h5 class="card-title"><%= photo.owner.username %></h5>
+        <p class="card-text"><%= photo.caption %></p>
+      </div>
+      <ul class="list-group list-group-flush">
+        <% photo.comments.each do |comment| %>
+          <li class="list-group-item">
+            <div class="d-flex">
+              <div class="flex-shrink-0">
+                <img src="..." alt="...">
```
Comment (APatel-AI) [inline app/views/photos/_photo.html.erb:14]: As a precaution to prevent errors you can also delete or add a default value for the image tag.

**Review by APatel-AI — APPROVED** [review]

**Review by AdolfoNava — APPROVED** [review]
Solid stuff everything seems to be fine

**Review by CarlosCastanedaDev — COMMENTED** [review]
Great job creating a partial for the tab info.

**Comment by CarlosCastanedaDev** [comment]
Please add the PR link to Canvas so I can grade it and you can move forward.

**Review by CarlosCastanedaDev — APPROVED** [review]
Great job, you can now merge these changes back to main.

**Pull request merged**